src/lwpMediaDevices.js

```diff
@@ -731,6 +779,43 @@ export default class extends lwpRenderer {
 
   /** Helper functions */
 
+  async _changeRingOutputDevice(preferedDevice) {
+    if (this._config.ringoutput.mediaElement.element) {
+      return this._config.ringoutput.mediaElement.element
+        .setSinkId(preferedDevice.id)
+        .then(() => {
+          this._availableDevices["ringoutput"].forEach((availableDevice) => {
```

Comment:
why not a `.map` to return the array with the updated values?

Reply:
I did it in that way following the same code guidelines of the project